src/context/WidgetSettingContext.tsx

```diff
@@ -159,7 +159,7 @@ export const WidgetSettingProvider = ({
             ...cachedSession,
             channelUrl: channel?.channelUrl ?? cachedSession.channelUrl,
           } satisfies WidgetSessionCache;
-          setWidgetSession(session);
+          setWidgetSession({ ...session, sessionToken });
```

Comment:
what about `onAutoCached`. No need to save sessionToken there?

Reply:
You can check detailed information from here: https://sendbird.atlassian.net/wiki/spaces/SDK/pages/2391277608/Widget+Session+management+strategy

Reply:
ah got it now, thanks. for auto, sessionToken is created in getWidgetSetting and cached in callback